src/software/networking/threaded_proto_multicast_sender.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <string>
+
+#include "software/networking/proto_multicast_sender.h"
```

Comment:
Also, by including this in the header, ProtoMutlicastSender will be exposed

Reply:
ah whoops meant to remove that. Will wait until above comment is resolved.